### DISCUSSION TITLE
Filtering target repositories based on environment variables

### DISCUSSION THREAD

**Discussion by larasika** [renovatebot/renovate#12170] [Request Help]
Hello, 

I know that we can filter target repositories with either directly listing the repos names in `RENOVATE_REPOSITORIES` variable or using the `--autodiscover-filter` flag with minimatch glob-style pattern. 

But is there any way we can filter the repos with extra filtering options ?

For example, I want to include all repos under `organizationName/groupName/` that have an environment `ENABLE_RENOVATE= TRUE` (or something like that) set in the project's environment variable. Therefore I will have to include `organizationName/groupName/*` in the `--autodiscover-filter` flag, but how can I tell Renovate to check on the environment variable ?

Clues and advises would be highly appreciated. Thank you !

**Comment by rarkins** [comment]
It's not supported today. First step would be to identify which API could be used to query that by Renovate.
